src/Pages/Profile/Profile.js

```diff
@@ -235,8 +241,8 @@ const Profile = () => {
           </Box>
         ) : (
           <Box>
-            <Typography sx={{ width: "400px" }}>Name: {user.name}</Typography>
-            <Typography sx={{ width: "400px" }}>Email: {user.email}</Typography>
+            <Typography>Name: {user.name}</Typography>
+            <Typography>Email: {user.email}</Typography>
             <Button
```

Comment:
There was no Error when I tried to edit the account while the server was down.

Reply:
Yes, It is correct @XeniyaDob because the edit account does not have an end-point. The endpoint is still not set up in the back end. It's not related to my task

Reply:
Ok, got it.